src/main/java/com/example/nasastealphoto/model/Camera.java

```diff
@@ -0,0 +1,50 @@
+package com.example.nasastealphoto.model;
+
+import static jakarta.persistence.CascadeType.PERSIST;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.CreationTimestamp;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "cameras")
+public class Camera {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "camera_id_gen")
```

Comment:
@romanovosad87 unless you really need to specify a `sequenceName` or `allocationSize`, you can omit this manual config of `@SequenceGenerator`.

Reply:
I was trying different options not to mention `sequenceName` or `allocationSize` but I get such exception. Maybe it is related to the fact that I use H2 database.
![image](https://github.com/romanovosad87/nasa/assets/114337016/a828445f-9cb7-42f7-806a-4b5f91840fcd)